modules/ROOT/pages/upgrading.adoc

```diff
@@ -234,32 +284,33 @@ If the view request exceeds the default timeout of 75s (which would be expected
 === Tidy-up
 Sync Gateway 2.0 will *not* automatically remove the previous design documents.
 
-Removal of the obsolete design documents is done via a call to the xref:rest-api:rest_api_admin.adoc#tag/Server/operation/post__post_upgrade[`_post_upgrade`] endpoint in Sync Gateway's Admin REST API.
+To remove the obsolete design documents, call the xref:rest-api:rest_api_admin.adoc#tag/Server/operation/post__post_upgrade[`_post_upgrade`] endpoint in Sync Gateway's Admin REST API.
 This endpoint can be run in preview mode (`?preview=true`) to see which design documents would be removed.
 
 
 == 1.x to 1.5
 
 From: *Sync Gateway 1.1, 1.2, 1.3, 1.4* +
 To:  *Sync Gateway 1.5* +
```

Comment:
Maybe track in a separate ticket initiative - but these versions are over 8 years old and really need removing from the docs.
Keeping them around introduces unnecessary maintenance burden like these changes below.

Reply:
Nope, shadow buckets are also very old feature and no longer present in the product.